src/stylesheets/abstracts/_mixins.scss

```diff
@@ -22,17 +22,8 @@
     height: 100%;
     object-fit: cover;
     z-index: -10;
-  }
-  @if $cover != none {
-    &:after {
-      content: '';
-      width: 100%;
-      height: 100%;
-      top: 0; right: 0; bottom: 0; left: 0;
-      position: absolute;
-      z-index: -9;
-      background-color: $cover;
-      opacity: $transparency;
+    @if $filter != none {
```

Comment:
You don't need this `if` statement, because `none` is default value of `filter`.

p.s. Well, it's not bad, because with this `if` this mixin will not break other filters which can already exist, but it's not our case.